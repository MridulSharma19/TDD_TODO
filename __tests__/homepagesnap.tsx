import Homepage from "@/components/homepage/homepage";
import { render } from "@testing-library/react";

it("renders homepage unchanged", () => {
    const dummyFunc = jest.fn();

    const { container } = render(
        <Homepage
            tasks={["Go to sleep", "Eat", "Read a book"]}
            name={"Mridul"}
            handleRemove={dummyFunc}
            handleAdd={dummyFunc}
            showAdd={false}
            handleClick={dummyFunc}
        />
    );
    expect(container).toMatchSnapshot();
});
