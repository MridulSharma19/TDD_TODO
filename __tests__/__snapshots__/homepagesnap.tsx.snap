// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div
    class="home-container"
  >
    <h1>
      TODO APP
    </h1>
    <h2>
      Hello 
      Mridul
    </h2>
    <div
      class="list-item"
    >
      <span>
        Go to sleep
      </span>
      <button
        class="red"
      >
        Remove
      </button>
    </div>
    <div
      class="list-item"
    >
      <span>
        Eat
      </span>
      <button
        class="red"
      >
        Remove
      </button>
    </div>
    <div
      class="list-item"
    >
      <span>
        Read a book
      </span>
      <button
        class="red"
      >
        Remove
      </button>
    </div>
    <button
      class=""
    >
      Add New
    </button>
  </div>
</div>
`;
